_posts/2017-09-01-the-scourge-of-silos.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title:  "The Scourge of Silos"
+date:   2017-09-01 04:48:58 +0000
+categories: bdd tdd devops pairing mobbing
+author: Hamish Tedeschi
+image: img/silos1.png
```

Comment:
Should we create sub-folders for the images?  Or not there yet?

Reply:
I had that thought as I was adding the images. I'm happy to re-jig

Reply:
Any content related comments?